lib/diagnostic.js

```diff
@@ -46,10 +46,12 @@ let scopes = [/* your response here */];
 // method and temporarily bind it (or, change `this`) to `alderaan` before
 // executing it.
 //
-let fireOnAlderaan = function () {
-  // your code here
+let fireOnAlderaan = function (deathStar, alderaan) {
+  console.log(deathStar.fire.bind(alderaan));
```

Comment:
You should use a return statement here in order to pass the tests. (Instead of a `console.log()`)
The problem with using `.bind()` here is that it sets the context, but it doesn't call the function immediately. 
The correct answer is 
```js
let fireOnAlderaan = function () {
  return deathStar.fire.call(alderaan);
};
```